Doc/library/sqlite3.rst

```diff
@@ -937,13 +946,25 @@ Connection objects
 
    .. method:: setlimit(category, limit, /)
 
-      Set a connection runtime limit. *category* is the limit category to be
-      set. *limit* is the new limit. If the new limit is a negative number, the
-      limit is unchanged.
-
+      Set a connection runtime limit.
       Attempts to increase a limit above its hard upper bound are silently
-      truncated to the hard upper bound. Regardless of whether or not the limit
-      was changed, the prior value of the limit is returned.
+      truncated to the hard upper bound.
+      Regardless of whether or not the limit was changed,
+      the prior value of the limit is returned.
+
+      :param category:
+          The `SQLite limit category`_ to be set.
+      :type category: int
+
+      :param limit:
+          The value of the new limit.
+          If negative, the current limit is unchanged.
+      :type limit: int
```

Comment:
```suggestion
      :param int limit: The value of the new limit.
          If negative, the current limit is unchanged.
```
I wonder if this works, even though it might be better to keep both lines indented under the `:param *:`

Reply:
I'd have both below the `:param` in this case.